CHANGES.md

```diff
@@ -124,7 +124,11 @@ Changes without direct rule impact:
 
 ## Version 4.0.0 - 2024-02-14
 
-Important changes:
+
+ Important changes:
+
+Backport fix for 3MU-240701-1 - catch invalid character in multipart headers via new rule 922130 (Ervin Hegedus, Felipe Zipitría)
+Backport fix for 3MU-240701-2 - prevent using backslash in file names from v4 - updated rule 920120 - pl1, 920121 - pl2 (Felipe Zipitria)
```

Comment:
You need to create a new section like the one for Version 3.3.5, but called 3.3.6 😄

Reply:
Here:
![image](https://github.com/user-attachments/assets/aa3069e4-a0a1-41af-9609-5fce6129fe90)